esrally/metrics.py

```diff
@@ -1203,7 +1203,7 @@ def format_dict(d):
         console.println("No recent races found.")
 
 
-def create_race(cfg, track, challenge):
+def create_race(cfg, track, challenge, track_revision=None):
```

Comment:
What about the team_revision? I didn't see it getting stored in metrics.

Reply:
It's stored as part of ClusterMetaInfo.

Reply:
I did a test run with the latest commit in this PR against an Elasticsearch metrics store and didn't see the team revision recorded.
e.g.:

![image](https://user-images.githubusercontent.com/1754575/62368644-df8d8a00-b535-11e9-8e36-b552182e9561.png)

Btw the particular class [ClusterMetaInfo](https://github.com/elastic/rally/blob/c65654f8bf635de1162f4d97cc153ae7296e539e/esrally/mechanic/mechanic.py#L47) has a comment before indicating it's used internally.